contracts/contracts/interfaces/morpho/ILens.sol

```diff
@@ -0,0 +1,296 @@
+
+    function getCurrentP2PSupplyIndex(address _poolToken)
+        external
+        view
+        returns (uint256);
+
+    function getCurrentP2PBorrowIndex(address _poolToken)
+        external
+        view
+        returns (uint256);
+
+    function getCurrentPoolIndexes(address _poolToken)
+        external
+        view
+        returns (
+            uint256 currentPoolSupplyIndex,
+            uint256 currentPoolBorrowIndex
+        );
+
+    function getIndexes(address _poolToken, bool _computeUpdatedIndexes)
+        external
+        view
+        returns (
+            uint256 p2pSupplyIndex,
+            uint256 p2pBorrowIndex,
+            uint256 poolSupplyIndex,
+            uint256 poolBorrowIndex
+        );
+
+    /// USERS ///
+
+    function getEnteredMarkets(address _user)
+        external
+        view
+        returns (address[] memory enteredMarkets);
+
+    function getUserHealthFactor(
+        address _user,
+        address[] calldata _updatedMarkets
+    ) external view returns (uint256);
+
+    function getUserBalanceStates(
+        address _user,
+        address[] calldata _updatedMarkets
+    )
+        external
+        view
+        returns (
+            uint256 collateralValue,
+            uint256 debtValue,
+            uint256 maxDebtValue
+        );
+
+    function getCurrentSupplyBalanceInOf(address _poolToken, address _user)
+        external
+        view
+        returns (
+            uint256 balanceOnPool,
+            uint256 balanceInP2P,
+            uint256 totalBalance
+        );
+
+    function getCurrentBorrowBalanceInOf(address _poolToken, address _user)
+        external
+        view
+        returns (
+            uint256 balanceOnPool,
+            uint256 balanceInP2P,
+            uint256 totalBalance
+        );
+
+    function getUserMaxCapacitiesForAsset(address _user, address _poolToken)
+        external
+        view
+        returns (uint256 withdrawable, uint256 borrowable);
+
+    function getUserHypotheticalBalanceStates(
+        address _user,
+        address _poolToken,
+        uint256 _withdrawnAmount,
+        uint256 _borrowedAmount
+    ) external view returns (uint256 debtValue, uint256 maxDebtValue);
+
+    function getUserLiquidityDataForAsset(
+        address _user,
+        address _poolToken,
+        bool _computeUpdatedIndexes,
+        ICompoundOracle _oracle
+    ) external view returns (Types.AssetLiquidityData memory assetData);
+
+    function isLiquidatable(address _user, address[] memory _updatedMarkets)
+        external
+        view
+        returns (bool);
+
+    function computeLiquidationRepayAmount(
+        address _user,
+        address _poolTokenBorrowed,
+        address _poolTokenCollateral,
+        address[] calldata _updatedMarkets
+    ) external view returns (uint256 toRepay);
+
+    /// RATES ///
+
+    function getAverageSupplyRatePerBlock(address _poolToken)
+        external
+        view
+        returns (uint256);
```

Comment:
```suggestion
        returns (
            uint256 avgSupplyRatePerBlock,
            uint256 p2pSupplyAmount,
            uint256 poolSupplyAmount
        );
```

Reply:
Just realized the `ILens` is outdated on our repository. Just submitted changes suggestions to have the correct Lens interface